lib/omni_auth/strategies/custom_okta.rb

```diff
@@ -53,13 +53,13 @@ def client_options
       alias oauth2_access_token access_token
 
       def access_token
-        if oauth2_access_token
-          ::OAuth2::AccessToken.new(client, oauth2_access_token.token, {
-                                      refresh_token: oauth2_access_token.refresh_token,
-                                      expires_in: oauth2_access_token.expires_in,
-                                      expires_at: oauth2_access_token.expires_at,
-                                    })
-        end
+        return unless oauth2_access_token
+
+        ::OAuth2::AccessToken.new(client, oauth2_access_token.token, {
+                                    refresh_token: oauth2_access_token.refresh_token,
+                                    expires_in: oauth2_access_token.expires_in,
+                                    expires_at: oauth2_access_token.expires_at,
+                                  })
```

Comment:
This feels like maybe it shouldn't be here.

Reply:
This was `if oauth2_access_token ...` and for some reason, RuboCop decided to yell about it now... Maybe it came in from a merge?

Reply:
@amichal , does this change look ok to you?

Reply:
Yeah this is rubocop trying to clean up the code i copy/pasted from the OKTA strategy at https://github.com/omniauth/omniauth-okta/blob/master/lib/omniauth/strategies/okta.rb. I dont think oauth2_access_token can ever be nil and if you look at the alias line above its a bit confused/silly but has been working fine

Reply:
See https://github.com/omniauth/omniauth-okta/pull/17 where i tried to make a tiny fix upstream in the gem before giving up and inlining the code. if we have time to revisit it this whole file can be further simplified.